packages/@lwc/perf-benchmarks/src/__benchmarks__/ssr/table-render-10k.benchmark.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import { renderComponent } from '@lwc/ssr-runtime';
+
+import { generateMarkup } from '@lwc/perf-benchmarks-components/dist/ssr/benchmark/table/table.js';
+import Store from '@lwc/perf-benchmarks-components/dist/ssr/benchmark/store/store.js';
+
+const SSR_MODE = 'asyncYield';
+
+benchmark(`ssr/table-v2/render/10k`, () => {
+    run(() => {
+        const store = new Store();
+        store.runLots();
+
+        return renderComponent(
+            'benchmark-table',
+            generateMarkup,
+            {
+                rows: store.data,
+            },
+            SSR_MODE
+        );
+    });
```

Comment:
It'd be neat to have individual entries for `asyncYield`, `async` and `sync`, so we'd have a continuous comparison. But maybe it is tricky to juggle all the compiled outputs, so totally understand.

Reply:
Yeah I figured it wasn't worth the effort since we're just going to pick something soon.